packages/core/platform/platform.js

```diff
@@ -150,13 +150,7 @@ const parseUserAgentLegacy = (userAgent) => {
 		}
 	}
 
-	if ('webos' === plat.platformName) {
-		deprecate({
-			name: plat.platformName,
-			message: 'Refer `@enact/webos`\'s `platform` for webOS specific information.',
-			until: '5.0.0'
-		});
-	} else if (!['chrome', 'safari', 'firefox'].includes(plat.platformName)) {
+	if (!['chrome', 'safari', 'firefox'].includes(plat.platformName)) {
```

Comment:
Could you please remove this if statement too?

Reply:
It seems like this log is not needed now. I removed